src/gateway/constants.ts

```diff
@@ -0,0 +1,107 @@
+export const GATEWAY_URL = 'wss://gateway.discord.gg/?v=8&encoding=json';

```

Comment:
As mentioned, please fetch the api for the gateway URL

Reply:
Done in [2215fef](https://github.com/LibraryofCode/discordts/pull/8/commits/2215fefe444094fe271c12214978c2dc1b94c70e)